samples/snippets/src/main/java/com/example/bigquerystorage/WriteCommittedStream.java

```diff
@@ -17,22 +17,34 @@
 package com.example.bigquerystorage;
 
 // [START bigquerystorage_jsonstreamwriter_committed]
+
 import com.google.api.core.ApiFuture;
+import com.google.api.core.ApiFutureCallback;
+import com.google.api.core.ApiFutures;
 import com.google.cloud.bigquery.storage.v1.AppendRowsResponse;
 import com.google.cloud.bigquery.storage.v1.BigQueryWriteClient;
 import com.google.cloud.bigquery.storage.v1.CreateWriteStreamRequest;
 import com.google.cloud.bigquery.storage.v1.FinalizeWriteStreamRequest;
 import com.google.cloud.bigquery.storage.v1.JsonStreamWriter;
 import com.google.cloud.bigquery.storage.v1.TableName;
 import com.google.cloud.bigquery.storage.v1.WriteStream;
+import com.google.common.util.concurrent.MoreExecutors;
 import com.google.protobuf.Descriptors.DescriptorValidationException;
 import java.io.IOException;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Phaser;
 import org.json.JSONArray;
 import org.json.JSONObject;
 
 public class WriteCommittedStream {
```

Comment:
This name is a little confusing, maybe name it to WriteCommittedStreamSample? I am not sure the impact of changing the sample's name.

Reply:
I think most of our samples don't include "Sample" in the name. e.g., [createTable](https://github.com/googleapis/java-bigquery/blob/0f19738e5e4229a52c8cc6b40d3bace40f8df747/samples/snippets/src/main/java/com/example/bigquery/CreateTable.java#L44).

Maybe WriteToCommittedStream?

Reply:
Make sense. Currently it sounded like a Stream. Will we just create a new file, switch the pointer and delete the old file?

Reply:
sgtm  I'll do this in a followup